lib/crazy_eight_game.rb

```diff
@@ -0,0 +1,161 @@
+require 'JSON'
```

Comment:
These lines shouldn't be necessary if you are running them from run.rb, since you are already requiring them there.